tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -1682,15 +1679,19 @@ describe('IssueList', function () {
         },
       },
     };
-    wrapper = mountWithTheme(<IssueListOverview {...props} />);
+
+    const {routerContext} = initializeOrg();
+    wrapper = mountWithTheme(<IssueListOverview {...props} />, routerContext);
```

Comment:
Route context needs to be set now that the Pagination will render (because of the below `pageLinks`)